families/identity/python/Dockerfile

```diff
@@ -1,49 +0,0 @@
-# Copyright 2018 Cargill Incorporated
```

Comment:
I think this delete should go in the remove python commit, as well as the `families/identity/python/sawtooth_identity/.keep` delete